README.md

```diff
@@ -471,7 +471,7 @@ If you have a macOS system, you can follow both the tutorial for iOS and for And
 As a first step,  copy the sample configuration for the app.
```

Comment:
```suggestion
As a first step, if you want to run the app in production mode, set the production configuration.
```
Could it be useful change also this line?

Reply:
Yes sure, you're right!